charts/prometheus-kafka-exporter/values.yaml

```
@@ -71,9 +71,10 @@ service:
   port: 9308
   labels: {}
   annotations: {}
-  # Specifies a custom nodePort for external use.
-  # Must be used with service.type=NodePort.
+  # Specifies a custom nodePort for external use. It must be used with service.type=NodePort.
```

The field need not be used as it is optional.

```suggestion
  # Specifies a custom nodePort for external use. Used with service.type=NodePort.
```